solidity/hardhat.config.ts

```diff
@@ -90,16 +90,17 @@ const config: HardhatUserConfig = {
           testConfig.nonStakingAccountsCount +
           testConfig.stakingRolesCount * testConfig.operatorsCount,
       },
-      tags: ["local"],
+      tags: ["allowStubs"],
       // we use higher gas price for tests to obtain more realistic results
       // for gas refund tests than when the default hardhat ~1 gwei gas price is
       // used
       gasPrice: 200000000000, // 200 gwei
+      allowUnlimitedContractSize: true,
```

Comment:
You won't need to set `allowUnlimitedContractSize` once https://github.com/keep-network/keep-core/pull/3057 is merged.